backend/src/controllers/change_request.js

```diff
@@ -125,8 +125,36 @@ const updateOne = async (request, reply) => {
   }
 };
 
+/**
+ * Add an item based on request body info.
+ *
+ * @param   {FastifyRequest} request FastifyRequest is an instance of the standard http or http2 request objects.
+ * @param   {FastifyReply}   reply   FastifyReply is an instance of the standard http or http2 reply types.
+ * @returns {object}
+ */
+const addOne = async (request, reply) => {
+  if (userCan(request, "change_request_add_one")) {
+    try {
+      const result = await Model.addOne(request.body);
```

Comment:
@ShawnTurple previously raised that we shouldn't be passing data directly from the response into the model.

It looks like all the validator code only has getOne routes. What's being done to validate/sanitize data?